### PR TITLE
Make publishable on npm

### PR DESCRIPTION
**Pull request by mdarens**
- remove dep on jquery
  - use `jquery-extends` package for `$.extends`
  - use native ES5 array methods instead of `$.inArray`

### CODE REVIEW

**Comment by chadly** [comment]
If you are going to kill `bower.json` completely, remove the UMD. Just convert it to CJS.

File: package.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
     "Google Maps",
     "map"
   ],
+  "scripts": {
+    "lint": "eslint src --ext=js --format=unix",
```
Comment (chadly) [inline package.json]: there is no `src` directory and the `--ext=js --format=unix` seem unnecessary

**Comment by chadly** [comment]
I'm just noticing, why does this repo have `bower_components` committed to it?

File: package.json (hunk 1 above)
Comment (mdarens) [inline package.json]: You are correct.
The format switch is to make file path detection work in the console.

**Comment by chadly** [comment]
Should we use `knockout-pinpoint` for the name? I know we alias it as `knockout.pinpoint` in our config.

**Comment by chadly** [comment]
Make sure to update the readme

File: README.md [hunk 2]
```diff
@@ -2,15 +2,13 @@
 
 > Knockout component to get coordinates/address based on map location
 
-View the [demo](http://civicsource.github.io/knockout-pinpoint/)
```
Comment (chadly) [inline README.md:5]: why you kill demo link?

Comment (mdarens) [inline README.md:5]: Because it depended on a github pages branch with bower dependencies

**Comment by chadly** [comment]
:shipit: :eggplant:

**Pull request merged**